README.md

```diff
@@ -1,70 +1,34 @@
-# Getting Started with Create React App
+# Reach Technical Test - Manchester Codes
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This is a technical test as part of Manchester Codes' Front End module using React. 
 
-## Available Scripts
+Based on a short brief, the objective of this task was to build a React web app that allows users to search for images that were returned from a query to NASA image API.
 
-In the project directory, you can run:
+This project uses:
 
-### `npm start`
+- React Testing Library
+- Jest for testing
+- Axios for retrieving data from the API
+- PropTypes to validate props
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+# Screenshot
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+![My Image](/public/tech-test-screenshot.png)
 
-### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Installation
 
-### `npm run build`
+- Clone this repository
+- cd into the repository
+- Run `npm install` to install dependencies
+- Run `npm start` to start the project in the browser
+- Run `npm test` to run associated tests
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## If I had more time..
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- add more styling to make search results and UX a bit cleaner
```

Comment:
Could add another point here to make the app responsive as there are a few inconsistencies when changing screen size